main.go

```diff
@@ -34,6 +35,13 @@ var (
 	k8sVersion   = "v1.18.15" // This should follow the version of k8s.io/kubernetes we are importing
 )
 
+func init() {
```

Comment:
Why an init function? This could perfectly well be done in `main()`)

Reply:
https://golang.org/doc/effective_go.html#init

> a common use of init functions is to verify or repair correctness of the program state before real execution begins.

To me, it clearly states failure should happen before any other step is taken.